dockerfile/README.md

```diff
@@ -1,10 +1,12 @@
 # Dockerfile for WSO2 Identity Server
+
 The Dockerfile defines the resources and instructions to build the Docker image for WSO2 Identity Server 5.3.0.
 
-## Build and run
+## How to build an image and run
+
 Follow below steps to build the WSO2 Identity Server 5.3.0 docker image and run in your local machine.
 
-The local copy of the `dockerfile` directory will be referred as `DOCKERFILE_HOME`.
+The local copy of the `Dockerfile` directory will be referred as `DOCKERFILE_HOME`.
```

Comment:
This should be "dockerfile", isn't that so?

Reply:
No, I think it should be `Dockerfile`. Refer its usage in [this](https://docs.docker.com/engine/reference/builder/) official documentation.

Reply:
In here we should be referring to the folder that we keep in the repo, not the Dockerfile.
Please refer to:  https://github.com/wso2/docker-ei/tree/master/dockerfiles